gundi_client_v2/client.py

```diff
@@ -36,10 +36,10 @@ async def post_observations(self, data: List[dict]) -> dict:
     async def post_events(self, data: List[dict]) -> dict:
         return await self._post_data(data=data, endpoint="events")
 
-    async def post_event_attachments(self, event_id: str, attachments: List[bytes]) -> dict:
```

Comment:
@vgarcia13 This looks like a breaking change, so we should go back and review any callers and make adjustments.
It might be only within the template repo and any forks.